### ISSUE TITLE
Manager/api status changes are not checked during health check 

### ISSUE THREAD

**Issue by jmv74211** [wazuh/wazuh-kibana-app#2910]
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.1.0  | 7.10.0     | 4101-1 |

Hello team,

When entering in wazuh app, if the manager or API is down, it is indicated in the initial health-check:

![image](https://user-images.githubusercontent.com/23462183/106713004-1399ad80-65fa-11eb-9dfc-ee1b8c6ba845.png)

but if we start the wazuh manager (checking that it has been connected successfully), the health-check status does not change, 
even though an animation is displayed indicating the opposite case.

![animation](https://user-images.githubusercontent.com/23462183/106713502-d84bae80-65fa-11eb-8abe-8ed877258126.gif)

Below I show the commented case with an example:

https://user-images.githubusercontent.com/23462183/106713690-1d6fe080-65fb-11eb-93ea-572b1c2b730f.mp4

It is proposed to check if the status of the manager or API has changed in which case update the health-check and continue to the wazuh app dashboard.

Best regards.

**Comment by frankeros** [comment]
Hi @jmv74211 
Currently, this is the expected behavior, but we take this as an enhancement for future releases. 

Thanks

**Comment by frankeros** [comment]
We need to control this error too 
![image](https://user-images.githubusercontent.com/26828184/112480036-5eab8380-8d54-11eb-8a65-589950fdc970.png)
It occurs when the wazuh-manager is not running
```
[root@wazuh-manager-master-0 /]# service wazuh-manager status
wazuh-clusterd: Process 3268 not used by Wazuh, removing...
wazuh-clusterd not running...
wazuh-modulesd is running...
ossec-monitord is running...
ossec-logcollector is running...
ossec-remoted is running...
ossec-syscheckd is running...
ossec-analysisd is running...
ossec-maild not running...
ossec-execd is running...
wazuh-db is running...
ossec-authd is running...
ossec-agentlessd not running...
ossec-integratord is running...
ossec-dbd not running...
ossec-csyslogd not running...
wazuh-apid is running..
```

**Comment by frankeros** [comment]
@CPAlejandro  please check if the PR #3109 also fix #2648
